run-tests.sh

```diff
@@ -1,29 +1,83 @@
 #!/bin/bash
 #
 # This file is part of REANA.
-# Copyright (C) 2017, 2018 CERN.
+# Copyright (C) 2017, 2018, 2019, 2020 CERN.
 #
 # REANA is free software; you can redistribute it and/or modify it
 # under the terms of the MIT License; see LICENSE file for more details.
 
+# Quit on errors
+set -o errexit
+
+# Quit on unbound symbols
+set -o nounset
+
+# Verify that db container is running before continuing
+_check_ready() {
+    RETRIES=40
+    while ! $2
+    do
+        echo "==> [INFO] Waiting for $1, $((RETRIES--)) remaining attempts..."
+        sleep 2
+        if [ $RETRIES -eq 0 ]
+        then
+            echo "==> [ERROR] Couldn't reach $1"
+            exit 1
+        fi
+    done
+}
+
+_db_check() {
+    docker exec --user postgres reana-postgres bash -c "pg_isready" &>/dev/null;
```

Comment:
`reana-postgres` : it would be nice to use the same container name as in `reana-dev ...` commands... Was it `postgres__reana-job-controller`?